pkgs/development/python-modules/surt/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, six
+, tldextract
+, pytestCheckHook
+, pytest-cov
+}:
+
+buildPythonPackage rec {
+  version = "0.3.1";
+  pname = "surt";
```

Comment:
```suggestion
  pname = "surt";
  version = "0.3.1";
```

Reply:
Thank you, I implemented your suggestion.